content/en/observability_pipelines/sources/splunk_hec.md

```diff
@@ -21,4 +23,26 @@ Select and set up this source when you [set up a pipeline][1]. The information b
 
 {{% observability_pipelines/log_source_configuration/splunk_hec %}}
 
+## Send logs from the Splunk Distribution of the OpenTelemetry Collector to Observability Pipelines
+
+To send logs from the Splunk Distribution of the OpenTelemetry Collector:
+
+1. Install the Splunk OpenTelemetry Collector based on the your environment:
+    - [Kubernetes][2]
+    - [Linux][3]
+2. Configure the Splunk OpenTelemetry Collector:
```

Comment:
I think we should swap steps two and three as seen here: https://github.com/DataDog/logs-psa-private/tree/main/POCs-Opps-hacks/splunk-otel-op#install-and-setup-op -- because you will need the IP Address / Load Balancer URL of OP before you can configure your Splunk OTel Collector.